docker/replacements.sed

```diff
@@ -0,0 +1,11 @@
+s#.*/(.*)\.(mp3)#music/\1\.\2#g
```

Comment:
this is a test file, move it to localdev. fail upon startup if the configured script file is not present